packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts

```diff
@@ -136,6 +118,9 @@ export class ForceFunctionStartExecutor extends SfdxCommandletExecutor<string> {
 
     const registeredStartedFunctionDisposable = FunctionService.instance.registerStartedFunction(
       {
+        rootDir: functionDirPath,
+        port: FUNCTION_DEFAULT_PORT,
+        debugPort: FUNCTION_DEFAULT_DEBUG_PORT,
```

Comment:
We could possibly support port other than default in the future. Currently we just use the default ports here.